### ISSUE TITLE
copy command too verbose

### ISSUE THREAD

**Issue by sedlund** [restic/restic#4033]
When there are 1000's of snapshots using the `copy` command to a remote  is quite chatty with this.

https://github.com/restic/restic/blob/32ffcd86a2a0d7013ff832d4ccc44a2f3ea5eff3/cmd/restic/cmd_copy.go#L134

Can this be changed to`Verboseff` ?

```diff
diff --git a/cmd/restic/cmd_copy.go b/cmd/restic/cmd_copy.go
index 08df7e8a..14ab1917 100644
--- a/cmd/restic/cmd_copy.go
+++ b/cmd/restic/cmd_copy.go
@@ -120,7 +120,6 @@ func runCopy(ctx context.Context, opts CopyOptions, gopts GlobalOptions, args []
        visitedTrees := restic.NewIDSet()

        for sn := range FindFilteredSnapshots(ctx, srcSnapshotLister, srcRepo, opts.Hosts, opts.Tags, opts.Paths, args) {
-               Verbosef("\nsnapshot %s of %v at %s)\n", sn.ID().Str(), sn.Paths, sn.Time)

                // check whether the destination has a snapshot with the same persistent ID which has similar snapshot fields
                srcOriginal := *sn.ID()
@@ -131,7 +130,8 @@ func runCopy(ctx context.Context, opts CopyOptions, gopts GlobalOptions, args []
                        isCopy := false
                        for _, originalSn := range originalSns {
                                if similarSnapshots(originalSn, sn) {
-                                       Verbosef("skipping source snapshot %s, was already copied to snapshot %s\n", sn.ID().Str(), originalSn.ID().Str())
+                                       Verboseff("\nsnapshot %s of %v at %s)\n", sn.ID().Str(), sn.Paths, sn.Time)
+                                       Verboseff("skipping source snapshot %s, was already copied to snapshot %s\n", sn.ID().Str(), originalSn.ID().Str())
                                        isCopy = true
                                        break
                                }
@@ -140,6 +140,7 @@ func runCopy(ctx context.Context, opts CopyOptions, gopts GlobalOptions, args []
                                continue
                        }
                }
+               Verbosef("\nsnapshot %s of %v at %s)\n", sn.ID().Str(), sn.Paths, sn.Time)
                Verbosef("  copy started, this may take a while...\n")
                if err := copyTree(ctx, srcRepo, dstRepo, visitedTrees, *sn.Tree, gopts.Quiet); err != nil {
                        return err

```

**Comment by MichaelEischer** [comment]
Sounds like a good idea. Would you mind opening a pull request?

**Comment by sedlund** [comment]
@MichaelEischer will do.  Thanks

**Comment by MichaelEischer** [comment]
Fixed by #4066.
